cpp/example_code/glue/README.md

```diff
@@ -0,0 +1,76 @@
+# AWS Glue code examples for the AWS SDK for C++
+
+## Overview
+
+Shows how to use the AWS SDK for C++ to manage AWS Glue resources.
+
+*AWS Glue is a serverless data integration service that makes it easier to discover, prepare, move, and integrate data from multiple sources for analytics, machine learning (ML), and application development.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
```

Comment:
You can also list the Single Action examples as they appear in the code library (even though they may all link to the same file).

Reply:
I am not sure about that, Rachel.  It is not consistent with what I have been doing.  I always consider the single action as standalone.  But I should list the api calls.

Reply:
Is the Single actions category missing? I'm checking against [the template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template).